src/main/java/me/arminb/sara/controller/models/CommentRequest.java

```diff
@@ -0,0 +1,45 @@
+package me.arminb.sara.controller.models;
+
+
+import me.arminb.sara.entities.Comment;
+import org.springframework.stereotype.Component;
+
+public class CommentRequest {
+
+    private String content;
+
+    private String user;
+
+    private String answerId;
```

Comment:
answerId will be passed through a path variable. I would remove it from this request class